### PR TITLE
add current `time` as a filename template option

### PR DESCRIPTION
**Pull request by dwoodiwiss**
And update the readme to reflect the new option.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you add an assert to the test here => https://github.com/sindresorhus/pageres/blob/master/test/test.js#L114

File: readme.md [hunk 1]
```diff
@@ -267,6 +267,7 @@ Available variables:
 - `height`: Height of the specified size, eg. `1000`
 - `crop`: Outputs `-cropped` when the crop option is true
 - `date`: The current date
+- `time`: The current time
```
Comment (sindresorhus) [inline readme.md]: Would be nice to show an example of how the time would look like.

**Comment by dwoodiwiss** [comment]
I've implemented those changes, but Travis CI seemed to have issues on Node.js: 0.10.

**Comment by kevva** [comment]
I think you can ignore Travis for now, but this needs a rebase.

**Comment by dwoodiwiss** [comment]
I have rebased my commits to one commit now.
I'm aware it says 'Commits: 3' though, is that an issue?

**Comment by sindresorhus** [comment]
You're still including 2 unrelated commits.

Easy fix: create a new branch, cherry-pick that one commit there, and force push to the remote branch of this PR.

**Comment by dwoodiwiss** [comment]
Cool, just cherry-pick(ed) the commit as you suggested. Thanks for the help!

**Comment by sindresorhus** [comment]
Excellent. Thanks :)

**Pull request merged**